File: routers/onboarding-router.js
import {
    health,
    questions
  } from "../controllers/onboarding-controller.js";
  import express from "express";
  
  export const onboardingRouter = express.Router();
  onboardingRouter.get("/onboarding/health", health);
  onboardingRouter.get("/onboarding/questions", questions);

File: internal/api/onboarding/onboarding_router.go
package onboarding

import (
	"aie/internal/middlewares"

	"github.com/gin-gonic/gin"
)

func OnboardingAPIs(router *gin.RouterGroup) {
	authRoute := router.Group("/onboarding/")
	{
		authRoute.GET("health", Health)
		authRoute.GET("questions", middlewares.AuthMiddleware(), Questions)
	}
}
